### PR TITLE
Update preferences help button URLs

### PR DESCRIPTION
**Pull request by jandubois**
The individual tabs will become separate doc pages, so should no longer be anchors but have their own path.

### CODE REVIEW

**Review by sunilarjun — APPROVED** [review]
All URL's for preferences are present and changes look good, thank you!

**Review by rak-phillip — APPROVED** [review]
LGTM - waiting for CI to complete before merging

**Pull request merged**
